packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -169,6 +169,20 @@ function fontFamilyFromSettings() {
 
 // 	return css;
 // }
+function useCSS(themeId: number): string {
+	const [css, setCSS] = useState('');
+
+	useMemo(() => {
+		const theme = themeStyle(themeId);
+		setCSS(`
+			:root {
+				background-color: ${theme.backgroundColor};
+			}
+		`);
+	}, [themeId]);
```

Comment:
Your code is correct but it does more than what is needed. If all you need is a derived value, the pattern is to do `return useMemo(() .....`, then inside the function `return ":root { ...."`. Then React will take of handling the state and so on. You don't need to manage state for this basically.

In fact I wouldn't create a custom effect for this (I'd just do `const css = useMemo...` but that's up to you.

Reply:
That should be done!